app/helpers/mustache_helper.rb

```diff
@@ -143,6 +143,15 @@ def page_config
     }
   end
 
+  def newsletter
+    {
+      form: {
+        action: "http://europeana.us3.list-manage.com/subscribe/post?u=ad318b7566f97eccc895e014e&amp;id=1d4f51a117",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.